roles/common/tasks/ssl.yml

```diff
@@ -56,7 +56,7 @@
     retries: 5
 
   - name: pip install requests
-    pip: name=requests version=2.11.1
+    pip: name=requests version=2.14.4
```

Comment:
Agreed. This does not need to be here. We will use the system packages and I'll revert this change.